package.json

```diff
@@ -49,11 +49,11 @@
     "@ngrx/router-store": "~10.0.0",
     "@ngrx/store": "~10.0.0",
     "@nguniversal/express-engine": "^10.1.0",
-    "@spartacus/assets": "3.0.0-rc.0",
-    "@spartacus/core": "3.0.0-rc.0",
-    "@spartacus/misc": "^3.0.0-rc.0",
-    "@spartacus/storefront": "3.0.0-rc.0",
-    "@spartacus/styles": "3.0.0-rc.0",
+    "@spartacus/assets": "3.0.0",
+    "@spartacus/core": "3.0.0",
+    "@spartacus/misc": "^3.0.0-rc.2",
```

Comment:
Some specific reason why misc lib is still with rc version?

Reply:
They haven't released higher version of misc lib yet...